<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>POCSAG Multichannel Realtime Decoder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>621</width>
      <height>311</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="addfreq_edit">
       <property name="placeholderText">
        <string>e.g. '100.025M'</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="freq_list">
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Console:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="freqCorrectionLabel">
       <property name="text">
        <string>Freq. Correction (ppm):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="freqcorr_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sampleRateLabel">
       <property name="text">
        <string>Sample Rate:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="samplerate_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="symbolRateLabel">
       <property name="text">
        <string>Symbol Rate:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="symrate_edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="centerFrequencyLabel">
       <property name="text">
        <string>Center Frequency:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="centerfreq_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>145</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="srcsink_check">
       <property name="text">
        <string>Source Sink</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="demodsink_check">
       <property name="text">
        <string>Demodulation Sink</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="debug_check">
       <property name="text">
        <string>Debug</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
